internal/resources/sample_bucket_schema.go

```diff
@@ -0,0 +1,42 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+)
+
+func SampleBucketSchema() schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"name":                       stringAttribute(required, requiresReplace),
+			"organization_id":            stringAttribute(required, requiresReplace),
+			"project_id":                 stringAttribute(required, requiresReplace),
+			"cluster_id":                 stringAttribute(required, requiresReplace),
+			"type":                       stringDefaultAttribute("couchbase", optional, computed, requiresReplace, useStateForUnknown),
+			"storage_backend":            stringAttribute(optional, computed, requiresReplace, useStateForUnknown),
+			"memory_allocation_in_mb":    int64DefaultAttribute(200, optional, computed, requiresReplace),
+			"bucket_conflict_resolution": stringDefaultAttribute("seqno", optional, computed, requiresReplace, useStateForUnknown),
+			"durability_level":           stringDefaultAttribute("none", optional, computed, requiresReplace),
+			"replicas":                   int64DefaultAttribute(1, optional, computed, requiresReplace),
+			"flush":                      boolDefaultAttribute(false, optional, computed, requiresReplace),
+			"time_to_live_in_seconds":    int64DefaultAttribute(0, optional, computed, requiresReplace),
+			"eviction_policy":            stringAttribute(optional, computed, requiresReplace, useStateForUnknown),
+			"stats": schema.SingleNestedAttribute{
+				Computed: true,
+				Attributes: map[string]schema.Attribute{
+					"item_count":         int64Attribute(computed),
+					"ops_per_second":     int64Attribute(computed),
+					"disk_used_in_mib":   int64Attribute(computed),
+					"memory_used_in_mib": int64Attribute(computed),
+				},
+			},
+		},
+	}
+}
```

Comment:
The Sample Bucket schema should not be exactly the same as the bucket schema because we only have one field during the create operation in the sample bucket.

Here, only the 'name,' 'organization_id,' 'project_id,' and 'cluster_id' fields should be required and  requireReplace, while all other fields should be computed.